packages/core/src/Stores/client-store.js

```diff
@@ -1053,18 +1053,18 @@ export default class ClientStore extends BaseStore {
                 ...response,
                 ...(is_maltainvest_account
                     ? {
-                        new_account_maltainvest: {
-                            ...response.new_account_maltainvest,
-                            currency,
-                        },
-                    }
+                          new_account_maltainvest: {
+                              ...response.new_account_maltainvest,
+                              currency,
+                          },
+                      }
                     : {}),
                 ...(is_samoa_account
                     ? {
-                        new_account_samoa: {
-                            currency,
-                        },
-                    }
+                          new_account_samoa: {
+                              currency,
+                          },
+                      }
```

Comment:
Are these prettier changes expected in this PR?

Reply:
think so. I had a conversation with Sandeep, he agreed with Matin and Yashim about Prettier changes in this PR.